### PR TITLE
Add work folder for training on Kubernetes Fest Tokyo 2020

### PR DESCRIPTION
**Pull request by atoato88**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

We will hold Kubernetes upstream training In a public event Kubernetes Fetst Tokyo 2020(formerly [Kubernetes Community Day Tokyo 2020](https://kubernetescommunitydays.org/events/2020-tokyo/)).
This PR adds work place folder for the event and some assets for training, and also OWNERS file.

**Which issue(s) this PR fixes**:

None

**Special notes for your reviewer**:

None

**Does this PR introduce a user-facing change?**:

```release-note
None
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

### CODE REVIEW

File: kft-2020/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Kubernetes Fest Tokyo 2020: Kubernetes Upstream Training
+
+<a href="https://github.com/kubernetes/kubernetes"><img src="https://github.com/kubernetes/kubernetes/blob/master/logo/logo.png" width="64"></a><a href="xxx"><img src="./images/xxx.png"></a>
```
Comment (ameukam) [inline kft-2020/README.md:3]: It's better to use the Markdown format : https://guides.github.com/features/mastering-markdown/#examples

Comment (atoato88) [reply kft-2020/README.md]: Thank you to comment.
I've fixed it.

File: kft-2020/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+## 資料 (Documents)
+
+* [参加者の皆様への宿題 Attendee prerequisites (in preparation)](docs/attendee-prerequisites.md)
+* [スライド Slides](docs/k8s-upstream-training-kft-2020.pdf)
+* [Kubernbetes contributors guide](https://github.com/kubernetes/community/tree/master/contributors/guide)
+
+## We got new contributors!! 
+
+To be done.
+![group photo](images/xxx.jpg)
```
Comment (ameukam) [inline kft-2020/README.md:19]: This image is not part of the PR.

File: kft-2020/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Kubernetes Fest Tokyo 2020: Kubernetes Upstream Training
+
+[<img src="https://github.com/kubernetes/kubernetes/raw/master/logo/logo.png" width="64">](https://github.com/kubernetes/kubernetes)
+[<img src="./images/xxx.png" width="64">](xxx)
```
Comment (ameukam) [inline kft-2020/README.md:4]: I'm not sure to understand but if you want to display the image you need to add it to the PR. 
The Markdown code will be : 
```
![xxx](images/xxx.png)
```

Comment (jberkus) [reply kft-2020/README.md:4]: Apparently the image doesn't exist yet.  But since it doesn't, it should be commented out.

Also ... I don't think that's valid markdown.

File: kft-2020/README.md (hunk 2 above)
Comment (atoato88) [reply kft-2020/README.md:19]: I'll fix this by comment out.

File: kft-2020/README.md (hunk 3 above)
Comment (atoato88) [reply kft-2020/README.md:4]: @ameukam @jberkus Thank you to comments!

**1. about "xxx"**

Yes, "xxx" is just a placeholder which doesn't exist, but it's hard to understand.
I'll comment out items which doesn't exist.

**2. markdown format to display images with hyperlink**

I change this section to display "plain image" and "event title with hyperlink".

File: kft-2020/README.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+-->
+
+Welcome to Kubernetes Upstream Training!!
+
+This is the location of our Kubernetes Fest Tokyo 2020: Kubernetes Upstream Training (June 13th 2020) activities.
+
+## 資料 (Documents)
+
+* [参加者の皆様への宿題 Attendee prerequisites (in preparation)](docs/attendee-prerequisites.md)
+* [スライド Slides](docs/k8s-upstream-training-kft-2020.pdf)
```
Comment (ameukam) [inline kft-2020/README.md:14]: @atoato88 Are you planning to add this document later ?

Comment (atoato88) [reply kft-2020/README.md:14]: @ameukam Yes, I'll add this document later. Should I comment out here?

Comment (ameukam) [reply kft-2020/README.md:14]: Just wanted to be sure about what you're planning to do later. :-)

**Comment by atoato88** [comment]
/assign @nikhita
Could you check this?
